README.md

```diff
@@ -26,15 +27,15 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 ```
 
 ```text
-Your answer goes here...
+data-ng-click creates the data in the API using ngResource
 ```
```

Comment:
Answer: The use of the data- tag allows for the code to be validated in an HTML validator.